### DISCUSSION TITLE
[Question]: Are "surveillance" drives recommened?

### DISCUSSION THREAD

**Discussion by dennismarwood** [blakeblackshear/frigate#10869] [Ask A Question] [answered]
### What is your question:

I am aware that surveillance hdd are typically recommend for NVR. But frigate ai should limit the writes to the disk to pertinent times. So most of the time it will analyze in memory and not write disk. I have not yet used frigate so I may be wrong in these assumptions.
I am buying new drives that I want to use for frigate _and_ for general back up. Should I spend the money on surveillance drives or greater capacity?

**Comment by NickM-27** [comment, marked as answer]
I just use standard or nas drives and haven't seen any issues.
